src/index.ts

```diff
@@ -148,11 +148,13 @@ function appendStyle() {
             }
 
             .${namespace}-chartlist-scrobbles .chartlist-album {
-                margin-top: 13px;
+                margin-top: 15px;
                 margin-bottom: -2px;
                 position: absolute;
-                left: 133.5px;
-                width: 182.41px;
+                left: 134px;
```

Comment:
Why did you round this up from `133.5px` to `134px`? Now it looks misaligned, at least in the browsers that I've tested this in (Chrome, Edge, Firefox).

This is what 133.5px looks like, when I change the track name and album name to `| TEST |`, and zoom in as much as possible:
![image](https://github.com/RudeySH/lastfm-bulk-edit/assets/3918757/a456aaa6-d351-4fb7-a615-d0f6c8211c0e)

And this is what 134px looks like:
![image](https://github.com/RudeySH/lastfm-bulk-edit/assets/3918757/f39035c3-6ce0-43bc-9d51-7c94aa28d768)

I added a red line to show that 134px will misalign the text:
![image](https://github.com/RudeySH/lastfm-bulk-edit/assets/3918757/7eaaef2b-b06e-4f89-b286-07ad1e7d5787)